src/nvim/lua/executor.c

```diff
@@ -741,7 +741,7 @@ static bool nlua_state_init(lua_State *const lstate) FUNC_ATTR_NONNULL_ALL
 
   // neovim version
   lua_pushcfunction(lstate, &nlua_nvim_version);
-  lua_setfield(lstate, -2, "version");
+  lua_setfield(lstate, -2, "_version");
```

Comment:
if it makes things easier in this PR, go ahead and delete `nlua_nvim_version` and move its impl into `version.lua`. It's trivial:

```
return vim.fn.api_info().version
```